### PR TITLE
Message tweak

### LINKED ISSUE #44: Message windows need css tweaks
<Do not use this to ask about lost codes or accounts>
<More info here: https://github.com/Authenticator-Extension/Authenticator/wiki/Lost-Codes>

Message windows should have a border and like everything else.

### PR DESCRIPTION
**Pull request by mymindstorm**
closes #44 

Not sure about the shadow.

### CODE REVIEW

**Comment by mymindstorm** [comment]
Here's the window at 0.25 opacity if you want to try that:
image

File: css/popup.css [hunk 1]
```diff
@@ -852,7 +853,7 @@ body {
     position: absolute;
     width: 100%;
     height: 100%;
-    background: rgba(0, 0, 0, 0.5);
+    background: rgba(0, 0, 0, 0);
```
Comment (Sneezry) [inline css/popup.css]: we can safely remove this line as the default background is transparent

**Review by Sneezry — APPROVED** [review]

**Comment by mymindstorm** [comment]
On the background what do you think transparency should be set to? 0.5 is
definitely too dark, but do you think that 0.25 would look better than none?

**Comment by Sneezry** [comment]
transparent is okay. `background: rgba(0, 0, 0, 0);` equals to removing `background` attribute, so we can just remove it from the code.

**Pull request merged**